### PR TITLE
Add monitoring warning to config page and 2018 to footer

### PR DESCRIPTION
**Pull request by sergioc**

### CODE REVIEW

File: src/ServicePulse.Host/app/js/views/configuration/configuration.html [hunk 1]
```diff
@@ -17,14 +17,21 @@ <h1>Configuration</h1>
...
             <div class="col-sm-12">
+
+                <div class="alert alert-warning">
+                    <i class="fa fa-warning"></i> <strong>Warning:</strong> The list of endpoints below only contains endpoints with the hearbeats plug-in installed. Toggling heartbeat monitoring on/off won't toggle <a href="https://docs.particular.net/monitoring/metrics/in-servicepulse" target="_blank">performance monitoring</a> <i class="fa fa-external-link fake-link"></i>
+                </div>
+
                 <div class="row box box-no-click table-header">
                     <div class="col-xs-3 col-sm-2 col-lg-1 no-side-padding">
-                        <p>Monitoring</p>
+                        <p>Heartbeats monitoring</p>
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/views/configuration/configuration.html]: This title feels clunky.

What about just `Heartbeats`?
Or perhaps even dropping the title completely given that the description explains the list below?

Comment (sergioc) [reply src/ServicePulse.Host/app/js/views/configuration/configuration.html]: I'm good with making it just `Heartbeats`. We should keep those headers to leave no room for doubts about what those toggles are for.

**Comment by sergioc** [comment]
I've renamed all titles and links for old monitoring screens to hearbeats to eliminate confusion with the new monitoring screens.

File: src/ServicePulse.Host/app/layout/footer.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
                     <div>
                         
                         <span>
-                            2017 © Particular Software
+                            2018 © Particular Software
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/layout/footer.html]: :tada:

File: src/ServicePulse.Host/app/js/app.constants.js [hunk 3]
```diff
@@ -6,7 +6,7 @@
         .constant('scConfig', {
             default_route: '/dashboard',
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:33533/']
```
Comment (WojcikMike) [inline src/ServicePulse.Host/app/js/app.constants.js]: What is the reason to change this URL? is it only local change?

Comment (sergioc) [reply src/ServicePulse.Host/app/js/app.constants.js]: Not meant to be changed, I probably forgot to revert the change before commiting.

**Pull request merged**